src/main/java/org/folio/edge/orders/MainVerticle.java

```diff
@@ -24,6 +23,7 @@ public Router defineRoutes() {
     router.route().handler(BodyHandler.create());
     router.route(HttpMethod.GET, "/admin/health").handler(this::handleHealthCheck);
     router.route(HttpMethod.GET, "/orders/validate").handler(validateHandler::handle);
+    router.route(HttpMethod.POST, "/orders/validate").handler(validateHandler::handle);
```

Comment:
I'm not a vertx expert, but I wonder if this creates 2 `Route`s, one for `GET` and one for `POST`, or if somehow vertx internally combines this into 1 `Route`. We could be explicit here by using the same `Route` and have it respond to both `GET` and `POST`. Something like:
`router.route(HttpMethod.GET, "/orders/validate").method(HttpMethod.POST).handler(validateHandler::handle);`
or
`router.route("/orders/validate").method(HttpMethod.GET).method(HttpMethod.POST).handler(validateHandler::handle);`
or something else...

Reply:
Done